docs/_articles/en/index.md

```diff
@@ -32,29 +32,22 @@ lang: en
 </div>
 <br>
 
-## Salesforce Code Analyzer Plug-in
+## Salesforce Code Analyzer
```

Comment:
The banners in this file will need to change to indicate that 2.x has been end-of-lifed and 3.x is the latest release.

Reply:
@jfeingold35 I agree. But I think we should roll out these doc changes now/soon, then have a separate PR closer to the GA date that updates the banners AND the install.md files.